### PR TITLE
Update metrics to include inner transactions.

### PR DESCRIPTION
**Pull request by winder**
## Summary

Update metrics to consider inner transactions.

## Test Plan

New unit test.

### CODE REVIEW

File: conduit/pipeline/pipeline.go [hunk 1]
```diff
@@ -366,17 +366,30 @@ func (p *pipelineImpl) Stop() {
...
-	metrics.ImportedTxnsPerBlock.Observe(float64(len(block.Payset)))
 	metrics.ImportedRoundGauge.Set(float64(block.Round()))
 	txnCountByType := make(map[string]int)
+	innerTxn := 0
 	for _, txn := range block.Payset {
 		txnCountByType[string(txn.Txn.Type)]++
+		innerTxn += numInnerTxn(txn.SignedTxnWithAD)
+	}
+	if innerTxn != 0 {
+		txnCountByType["inner"] = innerTxn
```
Comment (shiqizng) [inline conduit/pipeline/pipeline.go:387]: nit: it may be ok to record 0 inner txn count in this metric.

File: conduit/pipeline/pipeline_test.go [hunk 2]
```diff
@@ -815,3 +815,94 @@ func TestMetricPrefixApplied(t *testing.T) {
...
+						basicTxn(sdk.PaymentTx)),
+					basicTxn(sdk.PaymentTx),
+					basicTxn(sdk.PaymentTx)),
+			},
+		},
+	}
+
+	assert.Equal(t, 6, numInnerTxn(block.Payset[7].SignedTxnWithAD))
+
+	// was not able to get this part to work.
```
Comment (shiqizng) [inline conduit/pipeline/pipeline_test.go:869]: is this line still needed?

**Review by shiqizng — COMMENTED** [review]
looks good. just a commented line that looks out of place.

File: conduit/pipeline/pipeline_test.go (hunk 2 above)
Comment (winder) [reply conduit/pipeline/pipeline_test.go:869]:
```suggestion
```

**Review by shiqizng — APPROVED** [review]

**Pull request merged**